engine/runtime-language-epb/src/main/java/org/enso/interpreter/epb/node/ContextRewrapNode.java

```diff
@@ -92,7 +93,7 @@ long doWrappedLong(
       Object l,
       GuardedTruffleContext origin,
       GuardedTruffleContext target,
-      @CachedLibrary(limit = "5") InteropLibrary numbers) {
+      @Shared("interop") @CachedLibrary(limit = "5") InteropLibrary numbers) {
```

Comment:
Is it good idea to share `numbers` with `bools`?

Reply:
I am not sure about that. But in c98693780626cca8ee3ab4b41e0bc52a6c6598a4 I have reverted so that `bools` have their own `@Exclusive` InteropLibrary.